<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelChat</class>
 <widget class="QDialog" name="ChannelChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="sendResult_lbl">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>510</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Send message Result...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="channelname_led">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="send_pbn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>500</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QListWidget" name="message_lstwdgt">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>491</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="messagebar_led">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>460</y>
     <width>491</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Write a message...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
